eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/LightningMessageCodecs.scala

```diff
@@ -205,7 +205,7 @@ object LightningMessageCodecs {
 
   val txSignaturesCodec: Codec[TxSignatures] = (
     ("channelId" | bytes32) ::
-      ("txId" | sha256) ::
+      ("txHash" | sha256) ::
```

Comment:
This `sha256` codec is weird. Codecs are supposed to deal with types, otherwise we will declare one codec per possible usage of integers.

Reply:
I agree, it's a bit confusing since it's just a short-hand for `bytes32`, but was only added to make it a bit more clear that a field is actually the `sha256` hash of _something_ (but we don't know what at the type level). Whether we keep this or not is probably out of scope of this PR though...

Reply:
It would only make sense (from a consistent scodec usage pov) if we had a `Sha256` type (backed by a `ByteVector32`). But we don't. It's definitely outside of this PR 👍